src/main/webapp/view/transcript/transcript-seeker.js

```diff
@@ -114,5 +117,10 @@ export default class TranscriptSeeker {
     return transcriptLinePointer;
   }
 
-  // TODO: Add a method that adds the eventListeners.
+  /**
+   * Returns the eventController.
+   */
```

Comment:
This comment isn't useful. I would expand on what the event controller does.

Reply:
gotcha, updated!